packages/fast-components-class-name-contracts-msft/src/progress/index.ts

```diff
@@ -4,16 +4,51 @@ import { ProgressClassNameContract as BaseProgressClassNameContract } from "@mic
  * The class name contract for the progress component
  */
 export interface ProgressClassNameContract extends BaseProgressClassNameContract {
+    /**
+     * The root of the circular progress component
+     */
+    progressCircular?: string;
+
+    /**
+     * The progress size control modifier
+     */
+    progressCircular__control: string;
+
+    /**
+     * The progress size container modifier
+     */
+    progressCircular__container: string;
+
+    /**
+     * The progress size page modifier
+     */
+    progressCircular__page: string;
+
     /**
      * The progress value indicator
      */
     progress_valueIndicator?: string;
 
+    /**
+     * The circular progress value indicator
+     */
+    progressCircular_valueIndicator?: string;
+
+    /**
+     * The indeterminate circular progress value indicator
+     */
+    progressCircular_valueIndicator__indeterminate?: string;
+
     /**
      * The indeterminate progress indicator
      */
     progress_indicator?: string;
 
+    /**
+     * The indeterminate circular progress indicator
+     */
+    progressCircular_indicator?: string;

```

Comment:
You wouldn't need this one - again, just select for `progress__circular progress_indicator`